VirtualLibrary/Forms/Registration.cs

```diff
@@ -123,10 +121,19 @@ private void Button1_Click(object sender, EventArgs e)
 
         private void RegisterButton_Click(object sender, EventArgs e)
         {
-            _mUserPresenter.AddUser();
 
-            //UserInformationInXMLFiles xml = new UserInformationInXMLFiles(new DirectoryInfo(Application.StartupPath).Parent.Parent.FullName + "\\UserInformation\\", 5);
-            //xml.AddUser(faceImages, this);
+            m_userPresenter.AddUser();
+      
+            UserInformationInXMLFiles xml = new UserInformationInXMLFiles(new DirectoryInfo(Application.StartupPath).Parent.Parent.FullName + "\\UserInformation\\",
+                                                                                Constants.FaceImagesPerUser);
```

Comment:
Odd line formatting

Reply:
Noted. Planning on fixing it on next pull request